tests/main/dirs-not-shared-with-host/task.yaml

```diff
@@ -8,10 +8,11 @@ details: |
   consistent behaviour across various distributions.
 
 environment:
-    DIRECTORY/ssl: /etc/ssl
```

Comment:
Can I recommend adding a new test that shows what happens to the `/etc/ssl` directory?

Reply:
There is a test already `tests/main/ca-certs-for-snaps/task.yaml`